packages/services/src/configuration.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Logger } from "pino";
+
+import { CompetitionRepository } from "@recallnet/db/repositories/competition";
+
+/**
+ * Configuration Service
+ * Manages dynamic configuration settings loaded from the database
+ */
+export class ConfigurationService {
+  private competitionRepo: CompetitionRepository;
+  private logger: Logger;
+
+  constructor(competitionRepo: CompetitionRepository, logger: Logger) {
+    this.competitionRepo = competitionRepo;
+    this.logger = logger;
+  }
+  /**
+   * Load competition-specific settings and update global configuration
+   * This method updates the global features object with settings from the active competition
+   */
+  async loadCompetitionSettings(
```

Comment:
is this used anywhere?  I think this whole file might be dead code, in which case lets delete it